plugins/org.eclipse.elk.alg.libavoid/src/org/eclipse/elk/alg/libavoid/Libavoid.melk

```diff
@@ -1,5 +1,6 @@
 /*******************************************************************************
  * Copyright (c) 2016, 2022 Kiel University and others.
+                 2023 Primetals Technologies Austria GmbH
```

Comment:
Can you add the organization to the line above?
I.e.: 2016, 2023 Kiel University, Primetals Technologies Austria GmbH and others.